src/include/sof/bit.h

```diff
@@ -8,7 +8,12 @@
 #ifndef __SOF_BIT_H__
 #define __SOF_BIT_H__
 
+#if ASSEMBLY
 #define BIT(b)			(1 << (b))
```

Comment:
I'm suprised it works in our asm. not Every asm supports it.

Reply:
@lgirdwood I think that in kernel macros like this are just for C, how do they deal with ASM?

Reply:
This should also work for all pre-processed assembly as BIT() will just be turned into a number.

Reply:
@lgirdwood For xtensa only `(1 << x)` works, `(1U << x)`, is not, so I guess it's something asm-compiler-specific.